### DISCUSSION TITLE
Default home dir

### DISCUSSION THREAD

**Discussion by Mereana51** [drakkan/sftpgo#1139] [Q&A]
Hi everyone,

When we create a new user in webadmin and let local home dir empty, it create a folder in /srv/sftpgo/data/%username%
Is there a way to change this default directory like /data/%username% ?

BR,
Mereana.

**Comment by JonBendtsen** [comment]
Yes there is, look at the users_base_dir value in the json configuration file.

Also look at the [full-documentation](https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md), expand the `Configuration file` and then expand the `Data Provider`
